src/wp-includes/rest-api/endpoints/class-wp-rest-font-library-controller.php

```diff
@@ -0,0 +1,453 @@
+<?php
+/**
+ * Rest Font Library Controller.
+ *
+ * This file contains the class for the REST API Font Library Controller.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.4.0
+ */
+
+/**
+ * Font Library Controller class.
+ *
+ * @since 6.4.0
+ */
+class WP_REST_Font_Library_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 6.4.0
+	 */
+	public function __construct() {
+		$this->rest_base = 'fonts';
+		$this->namespace = 'wp/v2';
+	}
+
+	/**
+	 * Registers the routes for the objects of the controller.
+	 *
+	 * @since 6.4.0
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => array( $this, 'install_fonts' ),
+					'permission_callback' => array( $this, 'update_font_library_permissions_check' ),
+					'args'                => array(
+						'font_families' => array(
+							'required'          => true,
+							'type'              => 'string',
+							'validate_callback' => array( $this, 'validate_install_font_families' ),
+						),
+					),
+				),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::DELETABLE,
+					'callback'            => array( $this, 'uninstall_fonts' ),
+					'permission_callback' => array( $this, 'update_font_library_permissions_check' ),
+					'args'                => $this->uninstall_schema(),
+				),
+			)
+		);
+
```

Comment:
These should be combined into one `register_rest_route` call since they are part of the same namespace and route.